robottelo/api/utils.py

```diff
@@ -455,7 +455,9 @@ def create_role_permissions(role, permissions_types_names, search=None):  # prag
         entities.Filter(permission=permissions_entities, role=role, search=search).create()
 
 
-def wait_for_tasks(search_query, search_rate=1, max_tries=10, poll_rate=None, poll_timeout=None):
+def wait_for_tasks(
+    search_query, search_rate=1, max_tries=10, poll_rate=None, poll_timeout=None, must_succeed=True
+):
```

Comment:
@pondrejk we do have another `wait_for_tasks` in capsule_mixins which makes this helper method available with a destructive sat object, which could break due to your nailgun change, so could you update that as well?

And do you mind opening another PR later for the master branch as it was merged only with a single review?

Reply:
Here it is https://github.com/SatelliteQE/robottelo/pull/10657 Though the nailgun change is not breaking, the parameter has a default value and it doesn't have to be set by callers

Reply:
yeah, later I saw that, but I feel its good to have that update to test negative scenarios, thank you!